force-app/main/default/classes/selectors/tests/FundingRequestSelector_TEST.cls

```diff
@@ -0,0 +1,64 @@
+@isTest
```

Comment:
Can we add tests to verify that object and field permissions are respected in the selector? (Scott P or I can help you with this in the morning)